{"name": "TC1_001: Verify signup_with_invalid_data via UI", "status": "passed", "steps": [{"name": "Step 1: Signup to Contact List App with invalid Data", "status": "passed", "attachments": [{"name": "image", "source": "ee0ca1df-6263-499b-a3b3-63614efa3994-attachment.png", "type": "image/png"}], "start": 1736758238197, "stop": 1736758240685}, {"name": "Step 2: Verify Failed Signup", "status": "passed", "attachments": [{"name": "image", "source": "9025f92a-95c6-4102-a447-3a0d28bc1fe9-attachment.png", "type": "image/png"}], "start": 1736758240685, "stop": 1736758241763}], "attachments": [{"name": "log", "source": "a67d3d80-34fd-444b-a95d-a36cfc60a764-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "user", "value": "{'firstName': 'Sandrix', 'lastName': '', 'email': 'Jimenix@example.com', 'password': '12345qwert', 'result': 'User validation failed: lastName: Path `lastName` is required.'}"}], "start": 1736758238197, "stop": 1736758241763, "uuid": "e0299c59-186f-4cf6-9b61-e775c04574a2", "historyId": "39200c60cd5ee8a5c206aa7a9ebd1947", "testCaseId": "6509c706c67decc84e09c503a9c6463f", "fullName": "tests.ui.test_signup_via_ui.TestUserSignupFunctionality#test_signup_with_invalid_data_via_ui", "labels": [{"name": "parentSuite", "value": "Regression"}, {"name": "suite", "value": "User SignUp Functionality"}, {"name": "severity", "value": "normal"}, {"name": "subSuite", "value": "TC1_001: Verify signup_with_invalid_data via UI"}, {"name": "feature", "value": "User SignUp Functionality"}, {"name": "tag", "value": "tags('UI_SignUp_functionality')"}, {"name": "host", "value": "CHEQ-J7M1HB3"}, {"name": "thread", "value": "14480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.ui.test_signup_via_ui"}]}